### PR TITLE
docs: added copy button

### PR DESCRIPTION
**Pull request by the-r3aper7**
# Overview

# What is it?

image

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests / types / typos

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

**Comment by the-r3aper7** [comment]
I have created a eventlistener which is listening to whole body which i think is not optimal is there any other solution to this?
however when i am trying to do `document.getElementsByName` or `document.body.getElementsByClass` none of them is showing up? 

What should i do?

File: packages/docs/src/root.tsx [hunk 1]
```diff
@@ -16,6 +16,21 @@ export default component$(() => {
 
   useContextProvider(GlobalStore, store);
 
+  useVisibleTask$(({ cleanup }) => {
```
Comment (mhevery) [inline packages/docs/src/root.tsx:19]: Could we solve this without `useVisbelTask$` which forces code to run eagerly?

**Review by mhevery — CHANGES_REQUESTED** [review]

File: packages/docs/src/root.tsx (hunk 1 above)
Comment (the-r3aper7) [reply packages/docs/src/root.tsx:19]: i'll look into it

Comment (mhevery) [reply packages/docs/src/root.tsx:19]: I think you want `useOnDocument('click', $(...));` https://qwik.builder.io/tutorial/hooks/use-on/

**Comment by gioboa** [comment]
@the-r3aper7 are you still on this PR?

**Comment by the-r3aper7** [comment]
yeah i will work on it busy with college actually

**Comment by gioboa** [comment]
@the-r3aper7 thanks for the fast feedback. The best solution is close this one because there is also a conflict to solve.
You can create a new PR when you are free/ready, more then welcome to merge the future PR. 💪

**Pull request closed without merging**